frontend/src/pages/HomePage.tsx

```diff
@@ -1,12 +1,78 @@
+import { useEffect, useState } from 'react';
+import { Link } from 'wouter';
 import { navigate } from 'wouter/use-location';
+import clsx from 'clsx';
+
+import { fetchApi } from '../api';
 
 import PrimaryButton from '../components/PrimaryButton';
 
+type Document = {
+  id: string;
+  name: string;
+  audio_file: string | null;
+  created_at: string;
+  changed_at: string;
+};
+
 export default function HomePage() {
+  const [documents, setDocuments] = useState<Document[] | null>(null);
+
+  useEffect(() => {
+    (async () => {
+      const response = await fetchApi('v1/documents/', {});
+
+      if (response.ok) {
+        setDocuments(await response.json());
+      }
+    })();
+  }, []);
+
   return (
-    <div className="container mx-auto py-6">
-      <PrimaryButton onClick={() => navigate('/login')}>Login</PrimaryButton>{' '}
-      <PrimaryButton onClick={() => navigate('/new')}>New Document</PrimaryButton>
+    <div className="container mx-auto p-6">
+      <div className="mb-8 flex justify-between items-center">
+        <h1 className="text-xl font-bold">transcribee</h1>
+        <div>
+          <PrimaryButton onClick={() => navigate('/login')}>Login</PrimaryButton>{' '}
+          <PrimaryButton onClick={() => navigate('/new')}>New Document</PrimaryButton>
+        </div>
+      </div>
+      <ul
```

Comment:
Not necessarily in this PR, maybe only once they get more complex: Can we split the list & list items out to separate components?

Reply:
Yes, we should totally do that, when the item get's more details 👍